backend/protocol_rpc/endpoints.py

```
@@ -1069,6 +1099,7 @@
     consensus: ConsensusAlgorithm,
     consensus_service: ConsensusService,
     transactions_parser: TransactionParser,
+    sqlalchemy_db=None,  # Optional for backward compatibility
```

Ruff (0.12.2)

Unused function argument: `sqlalchemy_db`

(ARG001)